source/iostreams.tex

```diff
@@ -2633,7 +2633,7 @@
 the function signature
 \tcode{basic_ostream\& stream::operator\shl(ios_base\& (*)(ios_base\&))}
 to permit expressions of the form
-\tcode{cout \shl dec}
+\tcode{cout \shl{} dec}
```

Comment:
Here's an example of the index entry order difference I mentioned:
![diff](http://eel.is/indexdiff.png)
If that change is deemed acceptable, then I can submit a pull request to remove them altogether.